docs/journal/01-cold-storage.md

```diff
@@ -138,51 +156,50 @@ In this configuration, devices post event data to a Service Bus topic. Worker ro
 
 **Concerns:**
 
-- **Immaturity.** As with Event Hub, this is a new technology. The developers at Fabrikam need to invest time in learning its capabilities and how to use them.
-
 - **Event Metadata.** The event stream passed to Stream Analytics does not include the event metadata that is captured by Event Hub. This metadata can be valuable and can include information not available in the main payload of the event data. If this metadata is required, it might be necessary to connect directly to Event Hub and use an alternative approach to Stream Analytics.
 
-### Event Processing - Selected Technologies
 
-Fabrikam decided to use Event Hub and Stream Analytics to ingest and process event data. Although these technologies are very new, the scalability, reliability, and securability (the ability to verify and protect event information as it is received), and ease of use swung the decision; the maintenance and monitoring costs of this solution are far less than those concerned with using web and worker roles.
+### Event Hub + Apache Storm
 
-***TODO: Would be good to show table of costs, comparing web/work roles to EH, Stream Analytics***
+:construction: TBD 
 
-The simulator (and later on, the real devices) will send events directly to Event Hub. A Stream Analytics job will fetch the data for every event and send it to cold storage. The data for each event will be saved as a line-delimited JSON object for maximum interoperability.
 
-Fabrikam also decided to implement a secondary batch processor for handling events directly from Event Hub. This batch processor has access to the event metadata that is not available to Stream Analytics and can perform additional processing based on this information. The results are also directed towards cold storage.
+## Exploring solutions for cold storage
```

Comment:
I'm incline to separate this into its own journal entry: one for ingestion and one for long-term storage. I'm a bit divided on it though.
I can see us expanding both sections. Also, the ingestion topics are just as relevant to #165.